colibre/registration.py

```diff
@@ -118,3 +118,44 @@
     smhm.name = name
 
     setattr(self, f"stellar_mass_to_halo_mass_{aperture_size}_kpc", smhm)
+
+# iterate through available dust types
+dust_idx = 0
+total_dust_fraction = metal_mass_fraction_gas*0.
+while 1:
+    if not hasattr(catalogue.dust_mass_fractions, f"dust_{dust_idx}"):
+        break
+    total_dust_fraction += getattr(catalogue.dust_mass_fractions, f"dust_{dust_idx}")
+    dust_idx += 1
+
+
+total_dust_mass = total_dust_fraction * catalogue.apertures.mass_gas_30_kpc
+total_dust_mass.name = "$M_{\\rm dust}$"
+
+setattr(self, f"total_dust_masses", total_dust_mass)
+
+#species fraction properties
+gas_mass = catalogue.masses.m_gas
+H_frac = catalogue.element_mass_fractions.element_0
+gal_area = np.pi * catalogue.projected_apertures.projected_2_rhalfmass_star_30_kpc**2
```

Comment:
```
galaxy_radii = catalogue.projected_apertures.projected_1_rhalfmass_star_30_kpc
galaxy_area = np.pi * (galaxy_radii * galaxy_radii)
```
We should use aperture 1 here as that's what used for the galaxy size plot in the pipeline.